core/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "graph-sdk-core"
+edition = "2021"
+homepage = "https://spec.dsnp.org"
+license = "Apache-2.0"
+publish = false
+repository = "https://github.com/libertyDSNP/graph-sdk/"
+version = "0.0.0"
+
+[dependencies]
+wasm-bindgen = {version = "0.2.63", optional = true}
```

Comment:
do we want to include wasm dependencies right now or just wait and add whenever we wanted to add support for wasm?

Reply:
I would say keep it, ideally the rust code should be wasm  compatible and decorated accordingly
can come in later too, so no hard preferences